Source/Core/VideoCommon/Assets/MaterialAsset.cpp

```diff
@@ -28,7 +28,10 @@ bool ParsePropertyValue(const CustomAssetLibrary::AssetID& asset_id, MaterialPro
     }
   }
   break;
-  };
+
+  case MaterialProperty::Type::Type_Undefined:
```

Comment:
Sure.  But this one also goes away with #12240 😄

Reply:
Will drop the commit from this change then